caikit/runtime/wrappers/pydantic_wrapper.py

```diff
@@ -0,0 +1,162 @@
+# Copyright The Caikit Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""
+This module holds the Pydantic wrapping required by the REST server,
+capable of converting to and from Pydantic models to our DataObjects.
+"""
+# Standard
+from typing import Dict, List, Type, Union, get_args, get_type_hints
+import enum
+
+# Third Party
+import numpy as np
+import pydantic
```

Comment:
The only thing that _might_ be problematic about this is that depending on how we manage these dependencies (proactive vs reactive), `pydantic` _might_ end up becoming a required dependency when we intend to keep it in the `runtime-http` set. This makes me think that it might make sense to have a `runtime/http_server` module and put the current `http_server.py` plus this file in there so that it's clear that's the only module where `pydantic` and `fastapi` are required.